anabot/runtime/installation/welcome/language.py

```diff
@@ -10,7 +10,10 @@
 @handle_act('')
 def language_handler(element, app_node, local_node):
     lang = get_attr(element, "value")
-    gui_lang_search = getnode(local_node, node_type="text")
+    gui_lang_search = getno
```

Comment:
This doesn't look correct.

Reply:
Huh, this must have happened somehow when I modified the comment that was originally on the same line, just shorter. Thanks for pointing this out.